test/functional/conftest.py

```diff
@@ -3,20 +3,20 @@
 from __future__ import unicode_literals
 
 import re
+from test.functional.mock_box.box import Box
```

Comment:
Pylint/isort isn't aware by default that 'test...' would be a "first party" import. All it took was a declaration in setup.cfg